src/content_mapblock.cpp

```diff
@@ -172,6 +172,13 @@ static inline void getNeighborConnectingFace(v3s16 p, INodeDefManager *nodedef,
 		*neighbors |= v;
 }
 
+namespace
```

Comment:
Well because the anonymous namespace isn't used anywhere else.  If the rest of the code does something one way it's just best to go with the flow.

If you can make a solid argument why we should all dump static for anonymous namespace, then by all means make a PR for it that changes it everywhere.